includes/discovery/os/linux.inc.php

```diff
@@ -12,6 +12,8 @@
         } elseif (snmp_get($device, 'fwVersion.1', '-Osqnv', 'UBNT-AirFIBER-MIB', 'ubnt') !== false) {
             $os = 'airos-af';
         }
+    } elseif (snmp_get($device, 'EXTRAHOP-MIB::extrahopInfoVersionString', '-Osqnv') !== false) {
```

Comment:
I know the other gets are like this but can you shift the EXTRAHOP-MIB to the end of the function:

`snmp_get($device, 'extrahopInfoVersionString', '-Osqnv', 'EXTRAHOP-MIB')`